src/components/buttons/HdLoaderButton.vue

```diff
@@ -73,11 +73,11 @@
 import { getRandomInt, circleToPath } from 'homeday-blocks/src/services/utils';
 import debounce from 'lodash/debounce';
 
-const LOADING_STATE = Object.freeze({
```

Comment:
You can keep the `Object.freeze` here hehehe

Reply:
no need a constant is fine too